sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/helloworld_parallel_for_pipeline_job_output_name_version.yaml

```diff
@@ -0,0 +1,17 @@
+display_name: parallel_for_pipeline
+type: pipeline
+jobs:
+  parallel_body:
```

Comment:
add case that the node has this output

Reply:
I use `helloworld_pipeline_job_register_node_output_name_version.yaml` to test when node has this output.